front/src/app.js

```diff
@@ -37,7 +37,43 @@ const theme = createTheme({
 
 const App = () => {
   const exercice = {
-    content: '# Exercice 1',
+    content: '# Exercice 1 \n '+
+    ' Lorem ipsum dolor sit amet, '+
+    'consectetur adipiscing elit, sed do eiusmod tempor incididunt'+
+    ' ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis'+
+    ' nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo'+
+    ' consequat. Duis aute irure dolor in reprehenderit in voluptate velit'+
+    'esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat'+
+    'cupidatat non proident, sunt in culpa qui officia deserunt mollit anim'+
+    'id est laborum.' +
+    ' \n \n '+
+    ' Lorem ipsum dolor sit amet, '+
+    'consectetur adipiscing elit, sed do eiusmod tempor incididunt'+
+    ' ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis'+
+    ' nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo'+
+    ' consequat. Duis aute irure dolor in reprehenderit in voluptate velit'+
+    'esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat'+
+    'cupidatat non proident, sunt in culpa qui officia deserunt mollit anim'+
+    'id est laborum.' +
+    ' \n \n '+
+    ' Lorem ipsum dolor sit amet, '+
+    'consectetur adipiscing elit, sed do eiusmod tempor incididunt'+
+    ' ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis'+
+    ' nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo'+
+    ' consequat. Duis aute irure dolor in reprehenderit in voluptate velit'+
+    'esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat'+
+    'cupidatat non proident, sunt in culpa qui officia deserunt mollit anim'+
+    'id est laborum.' +
+    ' \n \n '+
+    ' Lorem ipsum dolor sit amet, '+
+    'consectetur adipiscing elit, sed do eiusmod tempor incididunt'+
+    ' ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis'+
+    ' nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo'+
+    ' consequat. Duis aute irure dolor in reprehenderit in voluptate velit'+
+    'esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat'+
```

Comment:
I know that it is for dev, but it has nothing to do here !!
The `app.js` file is not made for that. Ok for know, but please remove it very quickly.